code/modules/clothing/suits/cloaks.dm

```diff
@@ -18,43 +18,103 @@
 	flags_1 = NODROP_1
 	flags_inv = HIDEHAIR|HIDEEARS
 
-/obj/item/clothing/neck/cloak/suicide_act(mob/user)
+/obj/item/clothing/suit/hooded/cloak/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is strangling [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return(OXYLOSS)
 
-/obj/item/clothing/neck/cloak/hos
+/obj/item/clothing/suit/hooded/cloak/hos
```

Comment:
Please make sure I can carry the HoS egun!!
Please?

Reply:
o fuck lol i forgot to let him hold guns

Reply:
no

Reply:
^ he be holding guns

Reply:
<3